src/main/kotlin/nl/info/zac/notification/NotificationReceiver.kt

```diff
@@ -259,6 +262,10 @@ class NotificationReceiver @Inject constructor(
 
     @Suppress("TooGenericExceptionCaught")
     private fun handleInboxDocuments(notification: Notification) {
+        if (notification.channel == Channel.CALLBACK_URL_TEST) {
+            LOG.warning("Received test callback URL status notification")
```

Comment:
```suggestion
            LOG.info("Received a notification on the test channel")
```
And why warning level? INFO should be ok or no?

Reply:
Wanted to have this logged in all cases, but indeed that might be too much for a normally working system.

Reduced to info